hazelcast/src/test/java/com/hazelcast/jet/impl/JetClientInstanceImplTest.java

```diff
@@ -127,4 +129,29 @@ public void fileNameWithoutExtension() {
         Path jarPath2 = Paths.get("foo");
         assertThrows(JetException.class, () -> jetClientInstance.getFileNameWithoutExtension(jarPath2));
     }
+
+    @Test
+    public void validateParameterObject_NullJarPath() {
+        // Member is required to start the client
+        createHazelcastInstance();
+        JetClientInstanceImpl jetClientInstance = (JetClientInstanceImpl) createHazelcastClient().getJet();
+
+        SubmitJobParameters parameterObject = new SubmitJobParameters();
+        Assertions.assertThatThrownBy(() -> jetClientInstance.validateParameterObject(parameterObject))
```

Comment:
nit: static import for `assertThatThrownBy`

Reply:
Ok. Used static import